### PR TITLE
Add custom cod docs

### PR DESCRIPTION
**Pull request by dumbpreacher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # Welcome
 
-Welcome to the VerneMQ documentation! This is a reference guide for most of the available features and options of VerneMQ. The [Getting Started guide](getting-started.md) might be a good entry point.
+Welcome to the Gojek VerneMQ documentation! This project is a fork of the awesome ❤️  [MQTT broker](https://github.com/vernemq/vernemq) of the same name. We want to add some functionalities that we think can be useful for specific use-cases. This is a reference guide for most of the available features and options of VerneMQ. The [Getting Started guide](getting-started.md) might be a good entry point.
 
+## Summary of primary additions over core VerneMQ (1.11.0)
 
+We have added the following features over the core VerneMQ (1.11.0) features. These features have been added as a result of few drawbacks that we felt should be addressed better.
+- This project provides an option to disable persistence: [no op engine](configuration/noopengine.md)
```
Comment (dhruvjain99) [inline README.md:8]: Can we rename noopengine.md to no-op-engine.md?

File: SUMMARY.md [hunk 2]
```diff
@@ -2,13 +2,10 @@
 
 * [Welcome](README.md)
 * [Getting Started](getting-started.md)
-* [Downloads](https://vernemq.com/downloads)
-* [VerneMQ / MQTT Introduction](https://vernemq.com/intro)
+* [MQTT Introduction](Add link here)
```
Comment (dhruvjain99) [inline SUMMARY.md]: Link missing?

File: configuration/routing.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+description: Configure Routing store.
+---
+
+One of the problems that we tried to address was that of routing state not syncing when one of the nodes did was restarted. The subscription state used to build up on other nodes and the restarted node was unable to catch up to the other nodes. We finally achieved this using an external store(redis) for maintaining subscription state.
```
Comment (dhruvjain99) [inline configuration/routing.md]:
```suggestion
One of the problems that we tried to address was that of subscription state not syncing when one of the nodes restarted. The subscription state used to build up on other nodes and the restarted node was unable to catch up to the other nodes. We finally achieved this using an external store(redis) for maintaining a common subscription state for all the nodes. 
```

File: configuration/the-vernemq-conf-file.md [hunk 4]
```diff
@@ -3,16 +3,47 @@ description: >-
   A closer look at an example vernemq.conf file (Note: This is a
   work-in-progress section)
 ---
+[Here](Add plugins here)
```
Comment (dhruvjain99) [inline configuration/the-vernemq-conf-file.md]: Plugins missing?

**Review by dhruvjain99 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dumbpreacher) [reply README.md:8]: The forked project follows a naming convention where we do not use any delimiters within words.

File: configuration/routing.md (hunk 3 above)
Comment (dumbpreacher) [reply configuration/routing.md]: Sure.

File: configuration/the-vernemq-conf-file.md (hunk 4 above)
Comment (dumbpreacher) [reply configuration/the-vernemq-conf-file.md]: Added

**Pull request merged**